docs/unified-test-documentation/dasharo-compatibility/S30-fan-speed.md

```diff
@@ -0,0 +1,107 @@
+# Dasharo Compatibility: Fan speed
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+   [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+   [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+   [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+   [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+
+## FAN001.001 CPU fan
+
+**Test description**
+
+This test aims to verify that the CPU fan RPM value is available to read.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+1. Install `lm-sensors` on the DUT.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. In the terminal window run the following command to get the RPM value of the
+   CPU fan:
+
+    ```bash
+    sensors | grep "CPU fan"
+    ```
+
+    > If the command fails, make sure the `system76_acpi` driver is available on
```

Comment:
We should add this info to `Test setup` section.

Reply:
Replaced